dashboard/lib/pd/workshop_survey_constants.rb

```diff
@@ -13,6 +13,7 @@ module WorkshopSurveyConstants
       ACADEMIC_YEAR_4_CATEGORY = 'academic_year_4',
       ACADEMIC_YEAR_1_2_CATEGORY = 'academic_year_1_2',
       ACADEMIC_YEAR_3_4_CATEGORY = 'academic_year_3_4',
+      CSF_CATEGORY = 'csf',
```

Comment:
will we need separate categories for intro and deep dive workshops? I'm not sure exactly how categories are used. How did you decide this was the correct category to create?

Reply:
We use `category` and `name` to map surveys to JotForm ids in `JotFormBackedForm.get_form_id`. The mapping itself is a hash defined in `globals.yml` or `locals.yml`. 
From what I see in `globals.yml` in production, this mapping is simply a way to group form ids by context; there is no real constraint. We can group all existing and future csf surveys in `csf` category (until it gets too big).